### ISSUE TITLE
Network throughput far worse with Ethernet in my office than wifi

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#548]
Around https://github.com/130s/hut_10sqft/issues/547 I've been measuring speedtest.net result and computers via Ethernet is getting less than 10 Mbps while a cellphone via wifi is getting 200+ Mbps. Clearly there's bottleneck somewhere in my Ethernet chain.

Won't likely tackle before relocation.

**Comment by 130s** [comment]
(In Park Summit)

- Wifi is so much faster...1+% packet loss is bad but negligible?
- Eth setup in question is 10 times slower but almost 0 packet loss is good.

From wifi (on a small, kind of embedded laptop):
```
noodle@turtlebot2:~$ ping google.com                                                                    
PING google.com(2607:f8b0:4002:c02::64 (2607:f8b0:4002:c02::64)) 56 data bytes                
64 bytes from 2607:f8b0:4002:c02::64 (2607:f8b0:4002:c02::64): icmp_seq=5 ttl=107 time=34.6 ms
64 bytes from 2607:f8b0:4002:c02::64 (2607:f8b0:4002:c02::64): icmp_seq=6 ttl=107 time=12.5 ms
:
--- google.com ping statistics ---
248 packets transmitted, 244 received, 1.6129% packet loss, time 247443ms
rtt min/avg/max/mdev = 6.738/14.562/34.563/3.603 ms
```
From Mac 2014 via Ethernet via the setup in question:
```
$ ping google.com                                                                    
:
64 bytes from 74.125.21.113: icmp_seq=203 ttl=106 time=344.604 ms
64 bytes from 74.125.21.113: icmp_seq=204 ttl=106 time=305.205 ms
--- google.com ping statistics ---
262 packets transmitted, 261 packets received, 0.4% packet loss
round-trip min/avg/max/stddev = 206.441/324.298/442.364/35.961 ms
```
From an IPC on the same setup:
```
$ ping google.com                                                                    
:
64 bytes from yx-in-f102.1e100.net (64.233.177.102): icmp_seq=237 ttl=106 time=335 ms
64 bytes from yx-in-f102.1e100.net (64.233.177.102): icmp_seq=238 ttl=106 time=336 ms
:
--- google.com ping statistics ---
239 packets transmitted, 237 received, 0% packet loss, time 238321ms
rtt min/avg/max/mdev = 200.928/326.681/433.529/37.540 ms
```

**Comment by 130s** [comment]
(Moved to Richmond) Today wifi is much worse for some reason. Only thing I did is to attach a Powerline so that the broadband router's power is taken from the Powerline's "server" module (does it do anything regarding network strength??).

p50
```
$ ping 130s-serval.local
:
692 packets transmitted, 686 received, 0.867052% packet loss, time 692441ms
rtt min/avg/max/mdev = 3.281/315.152/3911.580/423.069 ms, pipe 4

$ ping google.com
608 packets transmitted, 374 received, 38.4868% packet loss, time 613027ms
rtt min/avg/max/mdev = 14.242/114.560/4054.085/306.116 ms, pipe 4
```

Serval
```
$ ping google.com
:
513 packets transmitted, 504 received, 1% packet loss, time 512714ms
rtt min/avg/max/mdev = 12.694/76.838/3605.268/210.299 ms, pipe 4
```

**Comment by 130s** [comment]
In Richmond I set up the Powerline and tested from different rooms with different distances to the Powerline source (broadband router).

- Wifi: 180 Mbps
- Powerline in the same room: 421 Mbps
- Powerline in the office room with a hub: 87 Mbps

(Powerline client: PLW400, hub: TL-SG105)

So Powerline is effective when it's close but it still de-gradates over the distance, kind of like the similar way Wifi does. I guess, Powerline can be still advantageous as it should be able to reach out longer distance than wifi (I haven't tested this theory though).

Office is where we need the fastest throughput at this house. I don't feel like degrading the speed in the office to lower than 10 Mbps when I know it can go to 400+ Mbps.

Anyways, to the problem in this ticket's title, only way I can get faster speed via Ethernet than wifi would be to shorten the distance b/w Powerline server and client, which I'm not sure is feasible in our home. Only way I can think of achieving it is having cable outlet closeby from the office, which I haven't checked yet (if I find it's feasible I might go with that).

Closing.
